Arm-Subsytem/Hand/hand.hpp

```diff
@@ -71,40 +73,53 @@ class Hand
     wrist_.PrintWristData();
   }
 
-  void HandleConcurrentMovement(float speed,
-                          float thumb_position,
-                          float pointer_position,
-                          float middle_position,
-                          float ring_position,
-                          float pinky_position,
-                          float roll_position,
-                          float pitch_position)
+  void HandleConcurrentMovement(
+                          MissionControlData::Finger finger_data,
+                          MissionControlData::Wrist wrist_data,
+                          float speed)
   {
     thumb_.SetSpeed(speed);
-    thumb_.SetPosition(thumb_position);
+    thumb_.SetPosition(finger_data.thumb_angle);
     pointer_.SetSpeed(speed);
-    pointer_.SetPosition(pointer_position);
+    pointer_.SetPosition(finger_data.pointer_angle);
     middle_.SetSpeed(speed);
-    middle_.SetPosition(middle_position);
+    middle_.SetPosition(finger_data.middle_angle);
     ring_.SetSpeed(speed);
-    ring_.SetPosition(ring_position);
+    ring_.SetPosition(finger_data.ring_angle);
     pinky_.SetSpeed(speed);
-    pinky_.SetPosition(pinky_position);
-    wrist_.HandleWristMovement(speed, roll_position, pitch_position);
+    pinky_.SetPosition(finger_data.pinky_angle);
+    wrist_.HandleWristMovement(speed, wrist_data.roll, wrist_data.pitch);
   }
 
   // The following two functions are here to allow the rover arm system to
   // directly control the roll and the pitch of the wrist this is mainly for
   // testing due to the different hand drive modes, when the drive modes are
   // removed this may be removed as well
-  void SetWristRollPosition(float speed, float roll_position)
+  void SetWristRollPosition(float wrist_roll, float speed)
   {
-    wrist_.SetRollPosition(speed, roll_position);
+    wrist_.SetRollPosition(wrist_roll, speed);
   }
 
-  void SetWristPitchPosition(float speed, float pitch_position)
+  void SetWristPitchPosition(float wrist_pitch, float speed)
   {
-    wrist_.SetRollPosition(speed, pitch_position);
+    wrist_.SetRollPosition(wrist_pitch, speed);
+  }
+
+  void HandleMovement(
+                      MissionControlData hand_data, float speed)
```

Comment:
```suggestion
  void HandleMovement(MissionControlData hand_data, float speed)
```
Run the formatter again on this file

Reply:
oh yah I was going to do that, just slipped my mind thanks.